core/src/components/menu/menu.tsx

```diff
@@ -25,6 +25,7 @@ const focusableQueryString =
 @Component({
   tag: 'ion-menu',
   styleUrls: {
+    base: 'menu.scss',
```

Comment:
Does this component also need the `mode` and `useBase` virtual props? It does have some per-mode styling.